bcache/bcache.go

```diff
@@ -81,7 +81,6 @@ type PeriodStats struct {
 	CacheHits           uint64
 	CacheMissCollisions uint64
 	CacheMisses         uint64
-	CacheReadaheads     uint64
```

Comment:
Hmm, do we want to remove this completely? Or detect the removed metric and provide a `nil`.

Reply:
You're right, I was thinking about this after submitting this patch too. This metric is obviously still available on kernel version < 5.13.

Reply:
@SuperQ I'm new to Golang (and not a dev by trade), but I think I was able to accomplish detecting when the metric is removed, but I wasn't able to get it work when returning nil (understandable because the return type is `unit64`).

Reply:
Looks great!